Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/common/task/YarnHistoryTask.java

```diff
@@ -40,29 +44,32 @@
 
     protected final IClusterDetail clusterDetail;
     protected final String path;
-    private final CredentialsProvider credentialsProvider =  new BasicCredentialsProvider();
-
+    private static final CredentialsProvider CREDENTIALS_PROVIDER =  new BasicCredentialsProvider();
```

Comment:
only one instance is required here since we will set the username/password in constructor